application/dependencies.json

```diff
@@ -13,7 +13,7 @@
   },
   {
     "name": "clue/stream-filter",
-    "version": "v1.7.0"
+    "version": "1.x-dev b27b3b4"
```

Comment:
I'm not familiar with this file or where these are coming from.

Reply:
Strange. I have no idea, but I'll do some digging.

Reply:
I think they were changed to `dev` when I used `"minimum-stability": "dev"` temporarily. `composer install` didn't fix it so I did `composer remove silinternational/yii2-sentry` and `composer require silinternational/yii2-sentry` and that seemed to work.